tests/test-sources/plugins/languages/texpresso.nix

```diff
@@ -0,0 +1,5 @@
+{
+  empty = {
+    plugins.texpresso.enable = true;
+  };
```

Comment:
To be exhaustive, there ought to be a non-default `texpressoPackage` test too. I don't think that's necessary though.

Reply:
Yes, this is not critical.